### PR TITLE
Git Signing API: Remove custom media type

### PR DESCRIPTION
**Pull request by palash25**
Fixes: #761

### CODE REVIEW

**Comment by palash25** [comment]
cc @gmlewis review requested :smiley:

File: github/git_commits.go [hunk 1]
```diff
@@ -68,8 +68,7 @@ func (s *GitService) GetCommit(ctx context.Context, owner string, repo string, s
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept headers when APIs fully launch.
-	req.Header.Set("Accept", mediaTypeGitSigningPreview)
+	req.Header.Set("Accept", mediaTypeV3)
```
Comment (gmlewis) [inline github/git_commits.go]: Thank you, @palash25.

Please note that here and everywhere else in this PR, `mediaTypeV3` does not need to be explicitly added, as that is handled in `github.go`. Please address this and I'll review again.

Comment (palash25) [reply github/git_commits.go]: Removed it in the latest commit @gmlewis

**Comment by gmlewis** [comment]
Thank you, @gauntface!
Merging then preparing new release.

**Pull request merged**